Moodlet/app/src/main/res/values/colors.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="colorPrimary">#008577</color>
-    <color name="colorPrimaryDark">#00574B</color>
-    <color name="colorAccent">#D81B60</color>
+    <color name="colorPrimary">#2699FB</color>
+    <color name="colorPrimaryDark">#141414</color>
+    <color name="colorAccent">#2699FB</color>
```

Comment:
The values for these colors have been set in PR #71. You might have some conflicts with the colors and would have to define new ones.

Reply:
Colors aligning with PR #71 + added temp colors for now